recipes/cpr/all/conanfile.py

```diff
@@ -45,8 +45,6 @@ def source(self):
 
     def requirements(self):
         self.requires("libcurl/7.67.0")
```

Comment:
```suggestion
        self.requires("libcurl/7.67.0")
        if self.options.with_openssl:
            self.output.warn("OpenSSL support is not implemented yet. https://github.com/whoshuu/cpr/issues/31")
```

Reply:
It is functional, but not stable, I would change the string to "OpenSSL support is not stable yet. https://github.com/whoshuu/cpr/issues/31"